practice_05.18.19_polymorphysm/Task1_Figures/Circle.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace practice_05._18._19_classes.Task1_Figures
+{
+    public class Circle : Figure
+    {
+        private double radius;
+
+        public Circle(double ordinate, double abscissa, double radius)
+            : base(ordinate, abscissa)
+        {
+            this.radius = radius;
+        }
+
+        public double Radius
+        {
+            get
+            {
+                return this.radius;
+            }
+            set
+            {
+                if (value > 0)
+                {
+                    this.radius = value;
+                }
+            }
+        }
+
+        public override void Draw()
+        {
+            Console.WriteLine($"Abscissa: {this.Abscissa}\nOrdinate: {this.Ordinate}\nThe radius of {this.GetType().Name}: {this.Radius}");
```

Comment:
Do you remember I suggested to use Environment.NewLine instead of '\n'?

Reply:
Unfortunately, I can't remember what it is... (((